packages/sdks/web-js-sdk/src/sdk/fedcm.ts

```diff
@@ -100,6 +100,8 @@ type OneTapInitialize = ({
 
 type PromptNotification = {
   isSkippedMoment: () => boolean;
+  isDismissedMoment: () => boolean;
+  getMomentType: () => string;
```

Comment:
Do you think we should use this instead of just including `isDismissedMoment`? @orius123 

Maybe we can do a switch based on the value from `getMomentType` instead like in my first commit?

<img width="750" alt="image" src="https://github.com/user-attachments/assets/9568170a-d0c5-40b9-8068-22850e71acf4" />